swap_meet/item.py

```diff
@@ -1,2 +1,23 @@
 class Item:
-    pass
+    def __init__(self, category=None, condition=0.0):
+        self.category=category if category is not None else ""
+        self.condition=condition
+    def __str__(self):
+        return f'Hello World!'
+    def condition_description(self, condition):
+        descriptions={
+            0:"Brand New",
+            0.5: "Like New",
+            1.0: "Lightly Used",
+            1.5: "Used",
+            2: "Normal Wear-and-Tear",
+            2.5: "Rough Around the Edges",
+            3: "Seen Better Days",
+            3.5: "Lightly Usable",
+            4: "Could Use for Parts",
+            4.5: "Why Are You Selling This",
+            5: "Hazardous Waste"
+        }
+        for rating in descriptions:
+            if rating == condition:
+                return descriptions[rating]
```

Comment:
This was a good way to check for conditions. You could also create a conditional based on ranges to save some lines of code. Here is a suggestion: 
```suggestion
        if self.condition > 4.0:
            print("Very good condition")
        elif 3.0 < self.condition <= 4.0:
            print("Pretty good condition")
        elif  2.0 < self.condition <= 3.0:
            print("Noticeable wear and tear")
        else:
            print("Terrible")
       
```